build/ci-scripts/script.sh

```diff
@@ -0,0 +1,7 @@
+#! /bin/bash
+
+set -ex
+
+cd ..
+echo $PWD
+php mediawiki/tests/phpunit/phpunit.php mediawiki/extensions/WikibaseReconcileEdit/tests
```

Comment:
We could consider splitting this into two kinds of tests: MediaWiki integration tests, which we run using `mediawiki/tests/phpunit/phpunit.php`, and unit tests, which we run using `mediawiki/vendor/bin/phpunit`. (Compare [T96458](https://phabricator.wikimedia.org/T96458).)

Reply:
Yeah sounds good, i guess we need to write one first :)